server/client_manager.py

```diff
@@ -905,6 +912,13 @@ def change_area(self, area, password=""):
                         f"[{self.id}] {self.showname} enters from [{old_area.id}] {old_area.name}{desc}",
                         "1",
                     )
+                    if self.area.area_manager.passing_msg == True:
+                        if showname == '':
+                            showname = self.char_name
+                        self.area.send_ic(
+                            None, '1', 1, "", "", f'~~[º{self.showname}º enters from º{old_area.name}º.]', 
```

Comment:
Miscellaneous thing you can do, but to make these msgs as unobtrusive as possible, you can preface them with 
`f'~~}}}[`
so they type out immediately, or
`f'~~}}[`
so the text types out really fast.

Reply:
Aight, sounds good to me